packages/template-retail-react-app/app/components/standard-login/index.jsx

```diff
@@ -50,6 +51,14 @@ const StandardLogin = ({
                         <SocialLogin form={form} idps={idps} />
                     </>
                 )}
+                <Button
+                    onClick={() => setShowPasswordView(false)}
+                    borderColor="gray.500"
+                    color="blue.600"
+                    variant="outline"
+                >
+                    <FormattedMessage defaultMessage="Back to Sign In Options" id="login_form.button.back" />
```

Comment:
Sorry testing this again and I realized that this StandardLogin component is rendered differently for each of the 4 passwordless/social combination.

In the case where passwordless is disabled, the password field is shown so there's no need to display the back button. 
<img width="515" alt="Screenshot 2025-01-17 at 2 48 34 PM" src="https://github.com/user-attachments/assets/e46cf64a-837b-485e-b321-62871b1359d6" />

I think we can simply make this button display when `hideEmail` is `true`